content/blog/2016/2016-11-16-blueocean-beta-declarative-pipeline-pipeline-editor.adoc

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Announcing the Blue Ocean beta, Declarative Pipeline and Pipeline Editor"
+tags:
+- blueocean
+- ux
+- pipeline
+- jenkinsworld
+author: i386
+---
+
+At Jenkins World on Wednesday 14th of September the Jenkins project was happy to
+introduce the beta release of Blue Ocean. Blue Ocean is the new user experience
+for Jenkins is built from the ground up to take advantage of Jenkins Pipeline
+and has rethought entirely the way that modern developers will use Jenkins.
+
+Blue Ocean is available today via the Jenkins update center for Jenkins users
+running 2.7.1 and above.
+
+=== Get the beta
+
+Just search for _BlueOcean beta_ in the plugin manager, install it,
```

Comment:
Is the plugin display name in the Update Center really "BlueOcean beta"? Ick. Also, the "BlueOcean" rather than "Blue Ocean" thing annoyed me. =) But my actual nit here is that it should be "in the Update Center", not "in the plugin manager", IMO.